gn2/wqflask/templates/gnqa_answer.html

```diff
@@ -93,20 +93,27 @@ <h3><b>{{ reference.pubmed[0].get('title') }}:</b></h3>
 {% block js %}
 
 <script>
+  function updateRatingHandler(target, responseObj, args){
+    let {status, response} = responseObj.xhr
+    if (status==200 && args == "upvote"){
+	  htmx.toggleClass(htmx.find('#upvote'), 'btn-success');
+	  htmx.removeClass(htmx.find("#downvote"), "btn-danger");
+      }
+      else if(status == 200 && args == "downvote") {
+          htmx.toggleClass(htmx.find('#downvote'), 'btn-danger');
+	  htmx.removeClass(htmx.find("#upvote"), "btn-success");
+      }
+      else {
+        alert(`Error occurred with status ${status} and Error ${response}` )
+}}
 var query = {{ query|tojson }};
 var answer = {{ answer|tojson }}
 var {task_id} = {{ task_id|tojson }}
-  htmx.on("#upvote", "click", function(evt){
+htmx.on("#upvote", "click", function(evt){
       vote_count = htmx.find(".btn-success") ? 0 : 1
-      htmx.ajax("POST", `/gnqna/rating/${task_id}/${vote_count}`, {target: "#rate", swap:"innerHTML",values: {'query': query, 'answer': answer}}).then(()=>{
-	  htmx.toggleClass(htmx.find('#upvote'), 'btn-success');
-	  htmx.removeClass(htmx.find("#downvote"), "btn-danger");
-})});
+    htmx.ajax("POST", `/gnqna/rating/${task_id}/${vote_count}`, {target: "#rate", handler: (target,obj)=> updateRatingHandler(target,obj,"upvote"), swap:"innerHTML",values: {'query': query, 'answer': answer}})});
```

Comment:
can we have these htmx calls span multiple lines. This helps with code-readablity e.g.
```javascript
    htmx.ajax("POST", `/gnqna/rating/${task_id}/${vote_count}`, 
          { target: "#rate",
             handler: (target, obj) => updateRatingHandler(target, obj, "upvote"),
             swap: "innerHTML",
             values: { 'query': query,
                            'answer': answer
                          }
           }
    );
```

Reply:
this commit fixes this: https://github.com/genenetwork/genenetwork2/pull/863/commits/f3843170bb0fe4cca17b529be61fc6ae438acfb6, On the issue of having too many braces I dont think
there is an alternative I have removed single line braces to improve readabilty.